artview/__init__.py

```diff
@@ -0,0 +1,18 @@
+"""
+ARTView:
+========
+
+"""
+
+from . import core
+from . import components
+from . import plugins
+from . import scripts 
+
+
+run = scripts.scripts['standard']
+
+
+if __name__ == "__main__":
+    import sys
+    print "haha"
```

Comment:
This is a neat way to handle this. I guess that we could also have different variants.
Something we should consider is a "light-weight" version in the future.